src/Core/src/Platform/Android/NativePicker.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Android.Content;
+using Android.Views;
+using Android.Widget;
+using Android.Runtime;
+using ARect = Android.Graphics.Rect;
+
+#if __ANDROID_29__
+using AndroidX.Core.Graphics.Drawable;
+#else
+using Android.Support.V4.Graphics.Drawable;
+#endif
```

Comment:
I think we are always using AndroidX now.

Reply:
Right, removed.